messenger/decoder.go

```diff
@@ -139,22 +139,36 @@ func (d *httpDecoder) run(res http.ResponseWriter) {
 		log.V(2).Infoln(d.idtag + "run: terminating")
 	}()
 
+	connCh := make(chan net.Conn, 1)
 	go func() {
+		con, ok := <-connCh
```

Comment:
i like to use "open" as the 2nd variable name on channel returns, makes the if statement read better